apps/rush-lib/src/scripts/install-run.ts

```diff
@@ -352,11 +352,18 @@ function installPackage(packageInstallFolder: string, name: string, version: str
 
 /**
  * Get the ".bin" path for the package.
+ * If on Windows and the path contains spaces, the returned path will be quoted.
+ * (Paths with spaces don't seem to cause trouble for child_process.spawn() on Mac.)
  */
 function getBinPath(packageInstallFolder: string, binName: string): string {
+  const isWindows: boolean = os.platform() === 'win32';
   const binFolderPath: string = path.resolve(packageInstallFolder, NODE_MODULES_FOLDER_NAME, '.bin');
-  const resolvedBinName: string = (os.platform() === 'win32') ? `${binName}.cmd` : binName;
-  return path.resolve(binFolderPath, resolvedBinName);
+  const resolvedBinName: string = isWindows ? `${binName}.cmd` : binName;
```

Comment:
`getBinPath()` is supposed to return a path.  A path should not contain shell escaping.  The quotation marks you are adding here are an encoding detail of a particular shell, so this isn't a good place to apply that logic.